src/components/CohortBuilder/index.js

```diff
@@ -50,6 +56,45 @@ const CohortBuilder = () => (
       const categoriesSqonUpdate = newSqon => {
         mergeSqonToActiveIndex(newSqon);
       };
+
+      const createNewSqonExcludingParticipants = participantIds => {
+        saveSet({
+          type: 'participant',
+          sqon: {
+            op: 'not-in',
+            content: {
+              field: 'kf_id',
+              value: participantIds,
+            },
+          },
+          userId: loggedInUser.egoId,
+          path: 'kf_id',
+          api: graphql(api),
+        })
+          .then(({ data }) => {
+            const newSqon = {
+              op: 'and',
+              content: [
+                activeSqonIndex,
+                {
+                  op: 'in',
```

Comment:
this should be `not-in`

Reply:
I don't see why both the set *and* the query including the set should use `not-in`, it sounds illogical to me.

Reply:
oh oops, bad comment again... corrected!